recipes/hunspell/all/test_package/conanfile.py

```diff
@@ -1,18 +1,26 @@
+from conan import ConanFile
+from conan.tools.build import can_run
+from conan.tools.cmake import cmake_layout, CMake
 import os
 
-from conans import ConanFile, CMake, tools
-
 
 class TestPackageConan(ConanFile):
     settings = "os", "arch", "compiler", "build_type"
-    generators = "cmake", "cmake_find_package_multi"
+    generators = "CMakeDeps", "CMakeToolchain"
+    test_type = "explicit"
+
+    def requirements(self):
+        self.requires(self.tested_reference_str)
+
+    def layout(self):
+        cmake_layout(self)
 
     def build(self):
         cmake = CMake(self)
         cmake.configure()
         cmake.build()
 
     def test(self):
-        if not tools.cross_building(self):
-            bin_path = os.path.join("bin", "test_package")
-            self.run(bin_path, run_environment=True)
+        if can_run(self):
+            bin_path = os.path.join(self.cpp.build.bindir, "test_package")
```

Comment:
Maybe?
```suggestion
            bin_path = os.path.join(self.cpp.build.bindir[0], "test_package")
```

See my pr: [here](https://github.com/conan-io/conan-center-index/pull/18829/files#diff-d37f3885bf913ef957677a9101f6732d0b5bff277f7c3f432abee2edd7f35f49R25)

Reply:
These are the same so either works... (you missed an s in your suggestion)